orca-peering/src/main/kotlin/com/netflix/spinnaker/orca/peering/MySqlRawAccess.kt

```diff
@@ -204,4 +210,16 @@ open class MySqlRawAccess(
 
     return persisted
   }
+
+  private fun <T> withRetry(action: () -> T): T {
```

Comment:
you can also do this with a retry annotation:
https://github.com/spinnaker/fiat/blob/64021e98f8d55c11a83149dc8aacdb854342c777/fiat-roles/src/main/java/com/netflix/spinnaker/fiat/providers/internal/Front50DataLoader.java#L39

And then make the options configurable/overridable via spring:
https://github.com/spinnaker/fiat/blob/64021e98f8d55c11a83149dc8aacdb854342c777/fiat-roles/src/main/resources/resilience4j-defaults.properties

Reply:
now that I say that, the annotation approach would change this from returning a function to executing the retry so never mind

Reply:
there was also some weird thing that @jonsie and I tried to figure out (the annotation wasn't working somewhere... river maybe?) anyway, it was driving me mad so I didn't want to chance it here :)